src/Php56/bootstrap.php

```diff
@@ -21,4 +21,16 @@ function hash_equals($knownString, $userInput) { return p\Php56::hash_equals($kn
 
         function ldap_escape($subject, $ignore = '', $flags = 0) { return p\Php56::ldap_escape($subject, $ignore, $flags); }
     }
+
+    // Missing functions in PHP 5.5.9 - affects 32 bit builds of Ubuntu 14.04LTS
```

Comment:
Just powered up my 14.04 / 32bit virtual machine...

`if (PHP_VERSION_ID === 50509 && PHP_INT_SIZE === 4) { ... }` works nicely for me.